glusterhealth/reports/glusterd-peer-disconnect.py

```diff
@@ -16,3 +18,20 @@ def report_check_errors_in_glusterd_log(ctx):
     if num_errors > 0:
         ctx.warning("peer disconnect num_errors in Glusterd log file", num_errors=num_errors)
 
+
+def report_peer_disconnect(ctx):
+	cmd = "gluster peer status"
+	try:
+		out = command_output(cmd)
+		peer_count = out.split("\n")[0].split(":")[1].strip()
+		peer_conn_count = out.count("(Connected)")
+		dis_count = int(peer_count) - int(peer_conn_count)
+		if 0 < dis_count:
+			ctx.notok("One or more peer/s in disconnected/rejected state", total_peer_count=int(peer_count), disconnected_rejected_count=dis_count)
```

Comment:
Additionally cli output can be logged, so that it helps to know which peer(s) in that state.

logging.warning("Peer status: \n" + out)